### ISSUE TITLE
Screen Reader: Reports: Record details and buttons are grouped, can't focus and activate separately

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77538]
## Action Performed:
Prerequisites:
- the user is signed in

1. Using iOS VoiceOver, open the application.
2. Once the 'Inbox' screen opens, swipe right to focus on the 'Reports' button at the bottom of the screen and double-tap to activate it.
3. Once the 'Reports' screen opens, swipe right to focus on the record tiles or details that are available on the screen.
4. Note how the elements are focused and announced.

## Expected Result:
Record details or information, and the button (e.g., view) should be treated as a separate, focusable element in each record tile. Screen readers should allow users to navigate to and interact with each control independently

## Actual Result:
All record details and associated buttons are grouped together as a single interactive element. The screen reader cannot move focus to each button individually, preventing users from accessing or activating the buttons separately.

## Area issue was found in:
Reports

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure the elements are not grouped with each other. Hide the parent container from the screen reader and make each child object accessible.
https://reactnative.dev/docs/accessibility#accessible

## Environment:
Apple iPhone 16 - iOS 18.2, Wi-Fi - Native (No mobile browser)

## Screenshots/Videos

<img width="327" height="652" alt="Image" src="https://github.com/user-attachments/assets/5693d0f4-8a11-4683-b44b-b541ab33eb06" />

https://github.com/user-attachments/assets/35ac2a15-5e55-4141-9802-94e62dc06bd0

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7007062
**Issue reported by:** NA
**Slack conversation:** NA
